scripts/f5-config.py

```diff
@@ -252,7 +278,7 @@
         'make_public': True,
         'hosts': []
     },
-    'nova_console': {
+    'nova_spice_console': {
```

Comment:
we can change it back, it was my understanding that we were going to eventually support both spice and vnc so having this might be a good idea anyway.